src/routes/item/index.js

```diff
@@ -57,9 +57,21 @@ const editItem = async function(req, res) {
   }
 }
 
+const deleteItem = async function(req, res) {
+  try {
+    const item = await Item.findById(req.params.id).exec()
+    item.delete()
```

Comment:
I have multiple questions for this line.

1. I am wondering if `.delete()` works.
Mongoos support findOneAndDelete, deleteOne, and remove.
I referred to [this](https://stackoverflow.com/questions/42715591/mongodb-difference-remove-vs-findoneanddelete-vs-deleteone)

2. What does `delete()` return? If it returns a Promise, we want to use `await`.

3. Before this line, we would need to check if we found the item. If it does, we want to early return with an error.

Reply:
I changed the `delete` function to `deleteOne`.